apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -2,16 +2,48 @@ import fetchJson from "@/charterafrica/utils/fetchJson";
 
 const BASE_DOCUMENTS_URL = "https://openafrica.net/api/3/action/";
 
-async function formatDatasets(data) {
-  const {
-    result: {
-      count,
-      facets: { tags: allTags, groups: allCountries },
-      results,
-    },
-  } = data || {};
+const PageSize = 10;
 
-  const formattedDatasets = results?.map((dataset) => {
+async function packageSearch(params) {
+  try {
+    const response = await fetchJson.get(
+      `${BASE_DOCUMENTS_URL}package_search`,
+      {
+        params,
+      }
+    );
+    return response;
+  } catch (error) {
+    return error;
+  }
+}
+
+function formatResources(resources, author) {
+  return resources?.map((resource) => {
+    const {
+      created: resourceCreated,
+      id,
+      last_modified: resourceUpdated,
+      url,
+      format,
+      name: resourceName,
+      description,
+    } = resource;
+    return {
+      author,
+      created: resourceCreated,
+      description,
+      format,
+      id,
+      name: resourceName,
+      updated: resourceUpdated,
```

Comment:
I'm completely lost here: Why are we renaming attributes e.g. `created` to `resourceCreated` on first pass and then rename them again back to the original names on returning e.g. `resourceCreated` to `created`?